Reference/Searching/Examine/Quick-Start/index.md

```diff
@@ -25,39 +23,78 @@ We will make it possible to 'search' on the _People_ page, by adding a search ba
 -->
 <div>
     <form action="@Model.Url()" method="get">
-        <input type="text" placeholder="Search" id="query" name="query" />
+        <input type="text" placeholder="Search" name="query"/>
         <button>Search</button>
     </form>
 </div>
 <div class="employee-grid">
 ...
 ```
 This will create a basic input field at the top of the page and make it post to the same people page when submitted along with the search term.
-### Retrieving the Search Term
+### Using route hijacking
```

Comment:
I think "Handling the search request" would be a better headline here - route hijacking is an implementation detail.